package.json

```diff
@@ -39,19 +41,18 @@
     "marked": "^4.0.12",
     "moment-timezone": "^0.5.35",
     "prop-types": "^15.8.1",
-    "react": "^16.14.0",
-    "react-beautiful-dnd": "^12.0.0",
+    "react": "^18.2.0",
+    "react-beautiful-dnd": "^13.1.1",
     "react-color": "^2.19.3",
-    "react-dom": "^16.14.0",
+    "react-dom": "^18.2.0",
     "react-hook-form": "^7.32.0",
     "react-icons": "^4.4.0",
     "react-input-mask": "^2.0.4",
     "react-markdown": "^8.0.0",
     "react-query": "^3.34.16",
     "react-router-dom": "^5.1.2",
     "react-select": "^5.4.0",
-    "react-simplemde-editor": "^4.1.0",
-    "react-toggle-button": "^2.2.0",
+    "react-simplemde-editor": "^5.2.0",
```

Comment:
Upgraded to latest, some breaking changes mentioned in https://www.npmjs.com/package/react-simplemde-editor#new-in-v5

```
New in v5
[breaking] Full rewrite to hooks. Means more reactive so, probably, less bugs related with updates. Minimum React version required >=16.8.2
[breaking] easymde now a peer dependency, please install it manually
[breaking] label prop has been removed
[breaking] SSR safe nets removed, please make sure to import it dynamically
[breaking] options shall be memoized to prevent new instances from being created on each render and other related to that bugs (more on that below)
[potentially-breaking] Forwards ref, so you can easily get access to div wrapper by using ref prop.
[potentially-breaking] Lots of bugs fixed, examples updated
[potentially-breaking] @babel/runtime helpers are no longer inlined but imported.
```

But from testing locally all is well